src/components/Dialog/SettingDialog/FileNameTemplateDialog.vue

```diff
@@ -78,50 +78,51 @@
 <script setup lang="ts">
 import { computed, ref, nextTick } from "vue";
 import { QInput } from "quasar";
-import { useStore } from "@/store";
-import {
-  buildAudioFileNameFromRawData,
-  DEFAULT_AUDIO_FILE_BASE_NAME_TEMPLATE,
-  replaceTagIdToTagString,
-  sanitizeFileName,
-} from "@/store/utility";
+import { replaceTagIdToTagString, sanitizeFileName } from "@/store/utility";
+import { UnreachableError } from "@/type/utility";
 
 const props = defineProps<{
   openDialog: boolean;
+  defaultTemplate: string;
+  availableTags: (keyof typeof replaceTagIdToTagString)[];
+  currentTemplate: string;
+  fileNameBuilder: (pattern: string) => string;
```

Comment:
ここちょっとややこしい気がするのと、パットどれが何かわからない気がするので、ドキュメントコメント足しておくのどうでしょう。
1行ずつぐらいで良さそう。とりあえず考えの足しになればと思って提案書いてみました。あまり強い意見ではないです。

```suggestion
  /** デフォルトのファイル名パターン */
  defaultTemplate: string;
  /** パターン置き換えに使えるタグ文字列のリスト */
  availableTags: (keyof typeof replaceTagIdToTagString)[];
  /** ソフトウェアで保存しているファイル名パターン */
  currentTemplate: string;
  /** ファイル名パターンからファイル名を作る関数 */
  fileNameBuilder: (pattern: string) => string;
```